docs/devices-api/more-info/webhooks.mdx

```diff
@@ -34,8 +34,14 @@ Example payload:
   "users": [
     {
       "userId": "951faef1-5cfd-464a-81f7-31f76edf309e",
-      "providers": ["garmin"],
-      "connectedProviders": ["fitbit", "garmin", "cronometer"]
+      "providers": ["tenovi"],
+      "newDevices": ["12345678-abcd-1234-abcd-1234567890ab"],
```

Comment:
If we were to follow the pattern used for `providers`, shouldn't we call this `devices`? Assuming that this field would also be used to report a device disconnection, `devices` would make more sense. But I don't know much about Tenovi's flow, so will leave it up to you, just a heads up.

Reply:
Yea, it makes sense to keep the field names consistent! Good catch